2022-05-03-announcements.md

```diff
@@ -0,0 +1,72 @@
+
+## CFDE Portal Data and Feature Update
+
+We are excited to announce a new release of the [Common Fund Data Ecosystem Portal](https://app.nih-cfde.org/)! This release features both a data and feature update!
+
+We are very excited to announce the addition of a new Common Fund group: Glycosciences! The goal of Glycosciences is to create new resources, tools, and methods to make the study of sugars more accessible to the broader research community.
+
+### Common Fund Programs on the CFDE Portal:
+
+- 4D Nucleome Data Coordination and Integration Center
+- Extracellular RNA Communication
+- The Gabriella Miller Kids First Pediatric Research Program
+- Genotype-Tissue Expression
+- Glycosciences
+- The Human BioMolecular Atlas Program
+- The Human Microbiome Project
+- Illuminating the Druggable Genome
+- Library of Integrated Network-based Cellular Signatures
+- Stimulating Peripheral Activity to Relieve Conditions
+- University of California San Diego Metabolomics Workbench
+
+### New CFDE Portal Features:
+
+- **Personal collections** can be created and exported. Users can create collections of data they are interested in and search within a collection.
+- New icons on the homepage allow users to start a search by **disease**, **gene**, or **compound**.
+- Users can now search for new types of data: **time series**, **proteins**, **compounds**, and **substances**. These can be searched for at the individual level or within a project or collection group. For example, users can search for the compound T-disaccharide in kidneys in a collection. An example of this search is [here](https://app.nih-cfde.org/chaise/recordset/#1/CFDE:collection/*::facets::N4IghgdgJiBcDaoDOB7ArgJwMYFM6JFU1wGscBPOEAZQH0sUMdaAzMLAFxAF8AaUAJYQARumj4QAYQBiAEQCiIXiAZNW7DrUhgOKALbl6jZm06sylALp9Q6DqLTiEUuYuWqTGrRB37D23QNzChBrZQgBGEt3AAsUAVwkfAAmAAYAdjDkdGw8BGziHAsqOgAHNGEsGJw9dU4efhAhByd4FwUlEHLK6trTTQY9UrEoWm6qmrrNFmKskDsWmGcZDuVx3qmjIZGt4cdRmZCwkAio2PjE-AAWAGYARjuANmtrIA@sort(combined_fact,nid)).
+
+### CFDE Portal Summary Facts:
+
+- The CFDE Search Portal now contains data from eleven different Common Fund Programs
+- Data types on the Portal include assays, DNA and RNA sequences, images, text reports, and more
+- Available data can be visualized on the “Preview the data” section of the homepage
+
+The CFDE Portal can be accessed at: [https://app.nih-cfde.org/](https://app.nih-cfde.org/)
+Portal documentation can be found here: [https://docs.nih-cfde.org/en/latest/about/portalguide/](https://docs.nih-cfde.org/en/latest/about/portalguide/) 
+
+Please direct any questions about the CFDE Portal to [support@cfde.atlassian.net](mailto:support@cfde.atlassian.net)
+
+## Working Group Updates
+
+| Working Group | Date | Time | Agenda |
+| ----------------- | ----- | ----- | --------- | 
+Anatomy | TBD | 9am PT / 12pm ET | [Anatomy WG](https://docs.google.com/document/d/1K5L9WllqaABbr4MGO21ogDELyvtpVrD31wbvSNhx6ys/edit?usp=sharing)
```

Comment:
can we remove the TBD ones?

Reply:
it stops my slack from spamming with people asking